components/atom/textarea/src/index.scss

```diff
@@ -4,7 +4,10 @@
 $short-num-lines: 5 !default;
 $long-num-lines: 7 !default;
 
-$pm-atom-textarea: $p-m !default;
+$p-atom-textarea: 2px !default;
```

Comment:
I think the default one for this should be `0`, as it was before.

Reply:
Then, better using `initial`.

Reply:
Finally we 2 decided to set it to 2px cuz initial is not returning us the desired behavior